Decred Wallet/Extensions/String.swift

```diff
@@ -29,3 +29,18 @@ extension String {
         return prefix(1).uppercased() + self.lowercased().dropFirst()
     }
 }
+
+protocol Localizable {
+    var getLocalizedString: String { get }
+}
+
+protocol XIBLocalizable {
+    var xibLocKey: String? { get set }
+}
+
+extension String: Localizable {
```

Comment:
You do not need a protocol to add an extension method. Re-check the need for this `Localizable`, also the `XIBLocalizable` protocol.

Reply:
self.menuTitleLabel.text = menuItem.rawValue.lowercased().getLocalizedString
Yes to get the localized enum value of the menu title